components/layout_2020/formatting_contexts.rs

```diff
@@ -230,6 +252,18 @@ impl NonReplacedFormattingContext {
                 containing_block_for_children,
                 containing_block,
             ),
+            NonReplacedFormattingContextContents::TaffyFlex(fc) => fc.layout(
+                layout_context,
+                positioning_context,
+                containing_block_for_children,
+                containing_block,
+            ),
+            NonReplacedFormattingContextContents::Grid(fc) => fc.layout(
+                layout_context,
+                positioning_context,
+                containing_block_for_children,
+                containing_block,
+            ),
```

Comment:
```suggestion
            NonReplacedFormattingContextContents::Grid(formatting_context) => formatting_context.layout(
                layout_context,
                positioning_context,
                containing_block_for_children,
                containing_block,
            ),
```

Reply:
This looks outdated to me. The existing code now matches the surrounding code (which I suspect it didn't when this suggestion was made). But let me know if you still want this change.